problems/isOdd.js

```diff
@@ -7,8 +7,14 @@
 
 */
 
-function isOdd() {
-
+function isOdd(num) {
+    let n = 4
+    if (num % 2 === 0){
+        return true;
+    } else {
+        return false;
```

Comment:
This will return true for even numbers and false for odd numbers. YOu're close to what you want, it just needs to be changed to num % 2 === 1